pulp_rpm/app/migrations/0045_modulemd_fields.py

```diff
@@ -21,35 +18,24 @@ def populate_new_fields(apps, schema_editor):
 
     # Fix issue #2786 if already tried to update to 3.18.1
     modulemds_to_update = []
-    for modulemd in Modulemd.objects.filter(snippet__startswith="b\'---"):
+    for modulemd in Modulemd.objects.filter(snippet__startswith="b'---"):
```

Comment:
What changed here?

Reply:
good catch, not intentionally formatted by `black` (in CI we skip check of migrations)